charts/rollups-node/script.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Script to convert values from rollups node TOML file to helm chart values YAML file with descriptions
+
+# Define file names
+TOML_FILE="Config.toml"
+JSON_FILE="Config.json"
+TEMPLATE_VALUES_FILE="values.yaml.tpl"
+VALUES_FILE="values.yaml"
+
+# Download the Config.toml file from the rollups-node repository
+wget https://raw.githubusercontent.com/cartesi/rollups-node/main/internal/config/generate/Config.toml
+
+# Convert TOML to JSON using yq
+yq -oj '.' "$TOML_FILE" > "$JSON_FILE"
```

Comment:
Did you tried to use only `yq` without requiring to transpose to JSON to use `jq`?

It would reduce the requirements of tools to run this, and also `yq` could possibly already make the conversion from TOML to YAML without having to go through JSON.

I'm assuming that `rq` have the need functionality for us.

Reply:
I have tried to use only the yq, but I've got lots of errors like:

```
parse error: Invalid numeric literal at line 1, column 8
  CARTESI_LOG_LEVEL:
parse error: Invalid numeric literal at line 1, column 8
    default: info
```

OR

```
invalid input text "_unsorted[]) | \"..."
```    

I think it's not capable to do filtering in `yq` as like as `jq` in `.jSON` format.
That's why I choose `jq` in the script.

Reply:
Also, I do lots of complex transformations from every part of json file to yaml file. like, transforming descriptions to the comments in the final yaml file also consider the new lines and ... . 

So, I've tested those conditions with both yq and jq. and jq could handle these.

Reply:
I still think `yq` can do the jobs and insist on us keeping the requirements smaller.

Also, `yq` (just like `jq`) supports defining the output schema so we don't need to have `echo` and `sed`.

Although it works, it's hard to understand what's going on here with all these different tools.

See: https://mikefarah.gitbook.io/yq/operators

We can even add comments in the YAML output with `yq`.

https://mikefarah.gitbook.io/yq/operators/comment-operators

Reply:
I'll check